cmd/kubectl-package/command/treecmd/tree.go

```diff
@@ -91,6 +91,11 @@ func (t *Tree) Run(ctx context.Context, out io.Writer) error {
 			return fmt.Errorf("%w: test template with name %s not found", cmdutil.ErrInvalidArgs, t.ConfigTestcase)
 		}
 	default:
+		rawConfig := pkg.PackageManifest.Test.Template[0].Context.Config.Raw
+
+		if err := json.Unmarshal(rawConfig, &config); err != nil {
+			return fmt.Errorf("unmarshal config from test template %s: %w", t.ConfigTestcase, err)
```

Comment:
This looks like a copy/paste message. Can it be update to remove the `t.ConfigTestCase` option?

Reply:
The `t.ConfigTestCase` branch picks a testcase with the given name while the default one picks the first one when no name is specified. So no, not really copy paste

Reply:
oh right, my bad